app/src/main/res/layout/activity_debt_list.xml

```diff
@@ -39,13 +39,23 @@
     </android.support.v7.widget.CardView>
 
     <android.support.design.widget.FloatingActionButton
-        android:id="@+id/floating_add_button"
+        android:id="@+id/floating_give_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom|end"
+        android:layout_marginEnd="@dimen/fab_margin"
+        android:layout_marginBottom="@dimen/give_fab_bottom_margin"
+        android:src="@drawable/ic_add_white_48dp" />
```

Comment:
If by any chance you're experiencing lack in material icons, here's the [google pack](https://github.com/google/material-design-icons.git) ;)